CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 * Added `--emulation` option for `rtest.py`
   * Unit tests can be run with `[--emulation|-e|--test|-t]=<test_name>`
 * Added `DeviceSelect::FlaggedIf` and its inplace overload.
+* Added CUB macros missing from hipCUB
```

Comment:
This needs a bit more detail. Listing the macros would be helpful for users. If the macros can't be listed because the list is to long, say, then generally saying which broad categories of macros or similar would be good.

```suggestion
* Added CUB macros missing from hipCUB.
```

Reply:
Added a list of macros.